metals/src/main/scala/scala/meta/internal/metals/InteractiveSemanticdbs.scala

```diff
@@ -176,7 +176,11 @@ final class InteractiveSemanticdbs(
         clientConfig.initialConfig.compilers.timeoutDelay,
         clientConfig.initialConfig.compilers.timeoutUnit
       )
-    val textDocument = s.TextDocument.parseFrom(bytes)
+    val textDocument = {
+      val doc = s.TextDocument.parseFrom(bytes)
+      if (doc.text.isEmpty()) doc.withText(text)
```

Comment:
Is this check required?
It looks like `SemanticdbTextDocumentProvider` sets up `text` value correctly.

Reply:
It's actually not serialized to bytes, which I think might be the default way to write it to file.